pydarn/plotting/power.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Cooper Ross Robertson, Summer Student 2020
+import matplotlib.pyplot as plt
+import numpy as np
+
+from typing import List
+
+import pydarn
+
+
+class Power():
+    """
+    Power class to plot SuperDARN RAWACF data power related.
+
+
+    Methods
+    -------
+    plot_interference
+    """
+    def _str_(self):
+
+        return "This class provides the following methods: \n"\
+                " - plot_interference()"
+
+    def plot_interference(cls, records: List[dict], beam_num: int,
```

Comment:
Put in default values this will help with issues like people not putting in all the values

Reply:
done